package-resources.yml

```diff
@@ -0,0 +1,55 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  creationTimestamp: null
+  name: kapp-controller.carvel.dev.0.0.0
+spec:
+  refName: kapp-controller.carvel.dev
+  releasedAt: null
+  licenses:
+  - Apache 2.0
+  template:
+    spec:
+      deploy:
+      - kapp: {}
+      fetch:
+      - git: {}
+      template:
+      - ytt:
+          paths:
+          - config
+          - config-release
+      - kbld: {}
+  valuesSchema:
+    openAPIv3: null
+  version: 0.0.0
+
+---
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: PackageMetadata
+metadata:
+  creationTimestamp: null
+  name: kapp-controller.carvel.dev
+spec:
+  displayName: kapp-controller
+  longDescription: "Provide CRDs to declaratively install, manage, and upgrade applications on a Kubernetes cluster"
+  shortDescription: "install, manage, and upgrade applications"
```

Comment:
I think we might be missing some extra info in the metadata:

```
  providerName: carvel.dev
  maintainers:
    - name: Carvel team (see www.carvel.dev for team members)
  categories:
    - "package management"
```

Reply:
Done